### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by ciremoussadia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 0.2.1 - 2021/01/14
+
+* Update dependencies.
```
Comment (blambeau) [inline CHANGELOG.md:3]: Add info that path must be >= 2.0, since it's a kind of broken API in practice.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.com/enspirit/talktome.svg?branch=master)](https://travis-ci.com/enspirit/talktome)
 # Talktome - Talk to users easily
```
Comment (blambeau) [inline README.md:2]: Please move the badge below the title.

File: lib/talktome/version.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@ module Talktome
   module Version
     MAJOR = 0
     MINOR = 2
-    TINY  = 0
+    TINY  = 1
```
Comment (blambeau) [inline lib/talktome/version.rb:5]: Bump to 0.3.0. Upgrading path to >= 2.0 requires a minor change at least.

**Review by blambeau — CHANGES_REQUESTED** [review]

File: lib/talktome/version.rb (hunk 3 above)
Comment (ciremoussadia) [reply lib/talktome/version.rb:5]: Ok

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# 0.3.0 - 2021/01/14
+
+* Update dependencies. Path version must be >= 2.0
+
+* Add travis to check for multi build matrices.
 # 0.2.0 - 2019/03/20
```
Comment (blambeau) [inline CHANGELOG.md:6]: New line here above.

**Review by blambeau — CHANGES_REQUESTED** [review]
@ciremoussadia cosmetic change then you can merge. Do you have the necessary permissions already?

**Comment by ciremoussadia** [comment]
> @ciremoussadia cosmetic change then you can merge. Do you have the necessary permissions already?

@blambeau No don't have necessary permissions.

**Pull request closed without merging**